tools/print-all/print-all

```diff
@@ -106,8 +106,8 @@ for fil in sh.git('ls-files', _iter=True):
     try:
         filetype = extensions[os.path.basename(fil).split('.')[1]]
     except (KeyError, IndexError):
-        with open(fil, 'rb') as f:
-            header = f.readline()
+        with open(fil, 'rb') as file:
+            header = file.readline()
```

Comment:
We usually avoid using `file` as a local variable, since it's a Python 2 keyword.  

Why is this change needed?